python/mxnet/numpy_op_fallback.py

```diff
@@ -49,6 +49,49 @@ def _register_helper(prop_cls):
     return _register_helper
 
 
+@use_np  # enforce np shape and array semantics for all the methods in this class
+class Sort(operator.CustomOp):
+    """Fallback to NumPy sort operator."""
+    def __init__(self, axis, kind):
+        super(Sort, self).__init__()
+        self._axis = axis
+        self._kind = kind
+
+    def forward(self, is_train, req, in_data, out_data, aux):
+        out = np.sort(in_data[0].asnumpy(), self._axis, self._kind)
+        self.assign(out_data[0], req[0], _mx_np.array(out, dtype=out.dtype, ctx=out_data[0].ctx))
+
+    def backward(self, req, out_grad, in_data, out_data, in_grad, aux):
+        raise NotImplementedError('Operator sort does not support gradient computation')
```

Comment:
Sort can be made to be differentiable. The gradient is to  pass back the gradient to the stored indices.

Reply:
I think it is suitable to use numpy.argsort to get the index in forward.